src/09_dictionaries.py

```diff
@@ -36,12 +36,27 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+waypoints.append({
+    "lat": 12,
+    "lon": -12,
+    "name": "a fourth place"
+})
+
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
 # YOUR CODE HERE
 
+waypoints[0] = {
+        "lat": 43,
+        "lon": -130,
+        "name": "not a real place"
+}
```

Comment:
Could also just change the specific keys you want to modify instead of assigning a whole new dictionary.
```
waypoints[0]["name"] = 'not a real place'
waypoints[0]["lon"] = -130
```